packages/website/src/components/package-metadata.tsx

```diff
@@ -26,7 +27,21 @@ const MetaDataEntryLabel = styled.div`
   flex-basis: 25%;
 `;
 
-function parseRepositoryUrl(repository: string, directory?: string) {
+const MetaDataEntryValue = styled.div`
+  flex-basis: 75%;
+  flex-wrap: wrap;
+`;
+
+const MetaDataDependency = styled.code`
+  display: inline-flex;
+  margin: 0 5px;
```

Comment:
Would be good to use `gridSize` from `@atlaskit/theme` for this to make sure our things remain consistently on-grid. Probably 0.5 * gridSize, or 4px

- [ ] use gridSize instead of custom pixel variable

Reply:
Updated